packages/manager/apps/web/client/app/private-database/order/clouddb/translations/Messages_fr_FR.json

```diff
@@ -31,6 +31,6 @@
   "private_database_order_clouddb_payment_get_checkout_error": "Une erreur est survenue lors de la récupération des informations de votre panier.",
   "private_database_order_clouddb_payment_get_payment_method_error": "Une erreur est survenue lors de la récupération des moyens de paiement.",
   "private_database_order_clouddb_payment_checkout_error": "Une erreur est survenue lors du paiement de votre panier.",
-  "private_database_order_clouddb_payment_checkout_success": "Votre commande pour votre base de données Cloud Databases a bien été prise en compte. Votre compte {{accountId}} a bien été débité de {{price}}. <a class=\"d-block\" target=\"_blank\" href=\"{{billUrl}}\" rel=\"noopener\">Voir ma facture</a>",
-  "private_database_order_clouddb_bill_success": "Votre <a target=\"_blank\" href=\"{{billUrl}}\" rel=\"noopener\">bon de commande</a> a été généré avec succès. Votre base de données Cloud Databases sera active après le paiement de celui-ci."
+  "private_database_order_clouddb_payment_checkout_success": "Votre commande pour votre base de données Web Cloud Databases a bien été prise en compte. Votre compte {{accountId}} a bien été débité de {{price}}. <a class=\"d-block\" target=\"_blank\" href=\"{{billUrl}}\" rel=\"noopener\">Voir ma facture</a>",
```

Comment:
**nitpick:** This sentence can be split into 2 and avoid html elements in the translation files.
```suggestion
  "private_database_order_clouddb_payment_checkout_success": "Votre commande pour votre base de données Web Cloud Databases a bien été prise en compte. Votre compte {{accountId}} a bien été débité de {{price}}.",
  "private_database_order_clouddb_payment_checkout_success_link": "Voir ma facture",
```
In HTML,

`<a class="d-block" target="_blank" href="{{billUrl}}" rel="noopener" data-translate="private_database_order_clouddb_payment_checkout_success_link"></a>`

Reply:
Hello, thanks for the review.
this translation is used in javascript. I'm not sure I can easily remove the html from translation

```
this.displaySuccessMessage(
            `${this.$translate.instant(
              'private_database_order_clouddb_payment_checkout_success',
              {
                accountId: this.defaultPaymentMean.label,
                price: prices.withTax.text,
                billUrl: url,
              },
            )}`,
          );

Reply:
Hello @lucas-chaigne,
You can still do it if you are using it in js.

`${$translate.instant('private_database_order_clouddb_payment_checkout_success)}<a class="d-block" target="_blank" href="{{billUrl}}" rel="noopener">${$translate.instant('private_database_order_clouddb_payment_checkout_success_link')}></a>`

Reply:
Hello, thanks. The requested change is made